src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -729,6 +779,7 @@ private static String executeRequestInternal(
     String theString;
     StringWriter writer = null;
     CloseableHttpResponse response = null;
+    Stopwatch stopwatch = Stopwatch.createStarted();
```

Comment:
do we want to use Stopwatch from guave or maybe we can easily implement similar class on our side?